apps/frontend/src/renderer/components/project-wizard/GitHubStep.tsx

```
@@ -0,0 +1,703 @@
+                  </h2>
+                  <p className="text-muted-foreground">
+                    {t('github.repoDescription')}
+                  </p>
+                </div>
+
+                {!repoAction ? (
+                  <div className="grid grid-cols-2 gap-4 mb-6">
+                    <button
+                      onClick={() => setRepoAction('create')}
+                      className="flex flex-col items-center gap-3 p-6 rounded-xl border-2 border-dashed hover:border-primary hover:bg-primary/5 transition-colors"
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)